app/src/main/java/net/squanchy/extensions/Booleans.kt

```diff
@@ -0,0 +1,7 @@
+package net.squanchy.extensions
+
+inline fun Boolean.whenTrue(block: () -> Unit) {
```

Comment:
❌ there is no need for this! `if` is a perfectly valid function that does the same thing.

We don't do these things here:

![Extension functions all the things](https://i.imgflip.com/1xoqek.jpg)